artemis-docker/prepare-docker.sh

```diff
@@ -181,10 +163,9 @@ if [ ! -d "${ARTEMIS_DIST_DIR}/docker" ]; then
   mkdir "${ARTEMIS_DIST_DIR}/docker"
 fi
 
-cp ./Dockerfile-* "$ARTEMIS_DIST_DIR/docker"
 cp ./docker-run.sh "$ARTEMIS_DIST_DIR/docker"
 
 echo "Docker file support files at:"
 tree "$ARTEMIS_DIST_DIR/docker"
```

Comment:
its still there

Reply:
What exactly is still there? tree has been removed.

Reply:
'tree' command in L188 will cause build error since not available in build stage. I propose to remove it like done in https://github.com/apache/activemq-artemis/pull/4453/files#diff-09d0fd7b380274547af0cdaf1cb41c22b3990dfed9848b560e9d9f77b97fb204L188
or replace with 'ls -l'